### ISSUE TITLE
Register Profile URI for AS2 RDF Profile?

### ISSUE THREAD

**Issue by dret** [jasnell/w3c-socialwg-activitystreams#236]
if AS2 is indeed using a profile URI (depending on #52), then maybe this could be done as well: http://www.iana.org/assignments/profile-uris/profile-uris.xhtml is a registry for profile URIs, created by @lanthaler via https://tools.ietf.org/html/rfc7284. maybe it might be appropriate to register a profile URI for the "AS2 RDF Profile"?

**Comment by jasnell** [comment]
There's no reason to have this as a separate issue. Let's keep the media type / profile discussion together in https://github.com/jasnell/w3c-socialwg-activitystreams/issues/52
